skaha/src/main/java/org/opencadc/skaha/image/GetAction.java

```diff
@@ -159,30 +160,36 @@ public List<Image> call() throws Exception {
                 if (f.isDone()) {
                     if (!imageList.isEmpty()) {
                         // collect the images from each project
-                       images.addAll(imageList);
+                        images.addAll(imageList);
                     }
                 }
             }
         } catch (InterruptedException | ExecutionException e) {
             log.error("Error when executing thread in ThreadPool: " + e.getMessage() + " caused by: " + e.getCause().toString());
-            throw e; 
+            throw e;
         } finally {
             if (taskExecutor != null) {
                 taskExecutor.shutdown();
             }
         }
-        
+
+        redis.put(IMAGE_LAST_UPDATED_TIMESTAMP, String.valueOf(System.currentTimeMillis()));
+        redis.put(PUBLIC_IMAGES, new ImageList(images));
```

Comment:
How does the ImageList class an improvement to List<Image>?  If there is an advantage then perhaps this method should return `ImageList` rather than `List<Image>`?